### PR TITLE
added entity, service, repository, controller of Employee

### PR DESCRIPTION
**Pull request by Myroslav-Polishchuk**

### CODE REVIEW

File: src/main/java/ua/danit/final_project/services/EmployeeService.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package ua.danit.final_project.services;
+
+
+import ua.danit.final_project.entities.Employee;
+import ua.danit.final_project.entities.User;
```
Comment (crystweb) [inline src/main/java/ua/danit/final_project/services/EmployeeService.java]: Code Smell Code Smell: Remove this unused import 'ua.danit.final_project.entities.User'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=39&issues=AWUu39d-W2AOP0Ck9j6j&open=AWUu39d-W2AOP0Ck9j6j)

[ISSUE_KEY]: AWUu39d-W2AOP0Ck9j6j

File: src/main/java/ua/danit/final_project/repositories/EmployeeRepo.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package ua.danit.final_project.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
```
Comment (crystweb) [inline src/main/java/ua/danit/final_project/repositories/EmployeeRepo.java]: Code Smell Code Smell: Remove this unused import 'org.springframework.data.jpa.repository.Query'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=39&issues=AWUu39d4W2AOP0Ck9j6i&open=AWUu39d4W2AOP0Ck9j6i)

[ISSUE_KEY]: AWUu39d4W2AOP0Ck9j6i

**Comment by Myroslav-Polishchuk** [comment]
[Add entity, repository, service and controller for employee](https://trello.com/c/bCWFsDGp/113-add-entity-repository-service-and-controller-for-employee)

File: src/main/java/ua/danit/final_project/controllers/EmployeeController.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  }
+
+  @GetMapping
+  public List<Employee> getAllEmployee() {
+    return employeeService.getAll();
+  }
+
+  @PostMapping
+  public Employee addEmployee(@RequestParam("uid") Long uid,
+                              @RequestParam("foname") String forename,
```
Comment (warfawl) [inline src/main/java/ua/danit/final_project/controllers/EmployeeController.java]: param `forename` - опечатка

File: src/main/java/ua/danit/final_project/entities/Employee.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.OneToOne;
+import javax.persistence.FetchType;
```
Comment (crystweb) [inline src/main/java/ua/danit/final_project/entities/Employee.java]: Code Smell Code Smell: Remove this unused import 'javax.persistence.FetchType'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=39&issues=AWU3cB7eW2AOP0CkE0NI&open=AWU3cB7eW2AOP0CkE0NI)

[ISSUE_KEY]: AWU3cB7eW2AOP0CkE0NI

File: src/main/java/ua/danit/final_project/entities/Employee.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.OneToOne;
+import javax.persistence.GenerationType;
```
Comment (crystweb) [inline src/main/java/ua/danit/final_project/entities/Employee.java]: Code Smell Code Smell: Remove this unused import 'javax.persistence.FetchType'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=39&issues=AWU3cB7eW2AOP0CkE0NI&open=AWU3cB7eW2AOP0CkE0NI)

[ISSUE_KEY]: AWU3cB7eW2AOP0CkE0NI

**Pull request merged**